packages/atomic/src/components/recommendations/atomic-recs-interface/atomic-recs-interface.tsx

```diff
@@ -43,13 +49,36 @@ export class AtomicRecsInterface
 {
   private store = createAtomicRecsStore();
   private commonInterfaceHelper: CommonAtomicInterfaceHelper<RecommendationEngine>;
+  private initialized = false;
 
   @Element() public host!: HTMLAtomicRecsInterfaceElement;
 
+  @State() public error?: Error;
+
+  /**
+   * The search interface [search hub](https://docs.coveo.com/en/1342/).
+   */
+  @Prop({reflect: true}) public searchHub = 'default';
+
+  /**
+   * The [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) identifier of the time zone to use to correctly interpret dates in the query expression, facets, and result items.
+   * By default, the timezone will be [guessed](https://day.js.org/docs/en/timezone/guessing-user-timezone).
+   *
+   * @example
+   * America/Montreal
+   */
+  @Prop({reflect: true}) public timezone?: string;
+
   /**
    * The recommendation interface headless engine.
    */
   @Prop({mutable: true}) public engine?: RecommendationEngine;
+
+  /**
+   * The search interface [query pipeline](https://docs.coveo.com/en/180/).
+   */
+  @Prop({reflect: true}) public pipeline?: string;
```

Comment:
From the docs https://docs.coveo.com/en/l25f0432/leverage-machine-learning/associate-a-content-recommendations-cr-model-with-a-query-pipeline it seems to be relevant ? We can add a note to use the recommendation property instead?

Reply:
Yeah the recommended (hah) way is to set a search hub + recommendation ID.

And the the query pipeline condition becomes `if searchhub=X and recommendation=Recommendation`